### PR TITLE
Image thumbnail

### PR DESCRIPTION
**Pull request by su-chang**
Related issue: https://github.com/wechaty/puppet-whatsapp/issues/165

### CODE REVIEW

File: src/manager.ts [hunk 1]
```diff
@@ -456,18 +456,38 @@ export class Manager extends EventEmitter {
...
+        return
+      }
+      await cacheManager.setMessageRawPayload(messageId, message)
+    }
   }
 
   private async onMessageAck (message: WhatsAppMessage) {
     logger.silly(`onMessageAck(${JSON.stringify(message)})`)
-    if (SkipTypeList.includes(message.type) && !message.hasMedia) {
+    await this.updateImageToCache(message)
```
Comment (bung87) [inline src/manager.ts:481]: Can you move `if (message.type === WhatsAppMessageType.IMAGE)` here? 

`onMessageAck` -> `await this.updateImageToCache(message)` is weird

**Review by bung87 — CHANGES_REQUESTED** [review]

File: src/manager.ts (hunk 1 above)
Comment (su-chang) [reply src/manager.ts:481]: It will write this judgement 3 times at least when use `updateImageToCache()`

Comment (su-chang) [reply src/manager.ts:481]: BTW, rename `updateImageToCache`  `createOrUpdateImageMessage`?

Comment (bung87) [reply src/manager.ts:481]: maybe name it to `possibleUpdateImageToCache` or something else ?

**Review by hcfw007 — APPROVED** [review]

**Pull request merged**